### DISCUSSION TITLE
Windows support status

### DISCUSSION THREAD

**Discussion by idkwhodatis** [ROCm/AMDMIGraphX#3169] [General]
Hi devs, 
I noted there are no pull request with windows label opened anymore. 
Does that mean Windows support would be officially added in the next release? thx

**Comment by causten** [comment]
MIGraphX via Windows will be via ONNX Runtime.  There is more to the ROCm stack then just MIGraphX but each PR is now functioning in CI

**Comment by PerfectFox265** [comment]
Do you if [Windows ML](https://learn.microsoft.com/en-us/windows/ai/new-windows-ml/supported-execution-providers) will add MIGraphX in their available execution providers ?
WindowsML is supposed to install the execution provider depending of the GPU of the user, but I don't see the point if AMD gpus are not supported.

**Comment by Artoriuz** [comment]
Any updates to this? https://pypi.org/project/onnxruntime-migraphx/#files is still only available for Linux.
